### PR TITLE
feature (sipi): adding sipi config and scripts

### LINKED ISSUE #404: Move Sipi configuration files to the Knora repository
I would like to move the Knora relevant Sipi configuration files to the Knora repository.

Which configuration files are needed? There are a few with Knora in the name:

Used for?
 - ``sipi.fake-knora-test-config.lua``

Used for?
 - ``sipi.init-no-knora-test.lua``
 - ``sipi.no-knora-test-config.lua`` 

Testing (serving leaves.jpg)
 - ``sipi.init-knora-test.lua``
 - ``sipi.knora-test-config.lua``
 
Production:
 - ``sipi.init-knora.lua``
 -  ``sipi.knora-config.lua``

### PR DESCRIPTION
**Pull request by subotic**
### Summary

- [x] Move any Knora specific configurations and scripts for sipi to this repository
- [x] Change Travis to use the sipi config and scripts from the knora repository.
- [x] Add Knora specific integration tests
- [x] Opened an issue for missing tests: https://github.com/dhlab-basel/Knora/issues/894. These tests are still exist in the `Sipi` repository, but before they are removed there, should be implemented here.

### Issues

- resolves #404

### CODE REVIEW

**Comment by benjamingeer** [comment]
Thanks, will review tomorrow.

File: docs/src/paradox/07-sipi/setup-sipi-for-knora.md [hunk 1]
```diff
@@ -23,16 +23,22 @@ License along with Knora.  If not, see <http://www.gnu.org/licenses/>.
...
-Once it is compiled, you can run Sipi with the following option:
-`./local/bin/sipi --config=config/sipi.knora-config.lua` (or
-`./local/bin/sipi --config=sipi.knora-test-config.lua` for using sipi
-for testing). Please see `sipi.knora-config.lua` for the settings like
-URL, port number etc. These settings need to be set accordingly in
-Knora's `application.conf`. If you use the default settings both in Sipi
-and Knora, there is no need to change these settings.
+```
+$ export DOCKERHOST=LOCAL_IP_ADDRESS
+$ docker run --rm -it --add-host webapihost:$DOCKERHOST -v $PWD/config:/sipi/config -v $PWD/scripts:/sipi/scripts -v /tmp:/tmp -v $HOME:$HOME -p 1024:1024 dhlabbasel/sipi:develop /sipi/local/bin/sipi --config=/sipi/config/sipi.knora-docker-config.lua
```
Comment (benjamingeer) [inline docs/src/paradox/07-sipi/setup-sipi-for-knora.md]: Is it normal that Sipi prints every log message twice?

```
$ docker run --rm -it --add-host webapihost:$DOCKERHOST -v $PWD/config:/sipi/config -v $PWD/scripts:/sipi/scripts -v /tmp:/tmp -v $HOME:$HOME -p 1024:1024 dhlabbasel/sipi:develop /sipi/local/bin/sipi --config=/sipi/config/sipi.knora-docker-config.lua

Sipi Version v1.2.0 Beta (build 2018-04-05 11:49)
Sipi Source Git-Version: 454d1d1
Subdir migration of /sipi/images/knora....
act_levels=0 levels=1
Sipi: Sipi Version v1.2.0 Beta (build 2018-04-05 11:49)
Sipi: Sipi Version v1.2.0 Beta (build 2018-04-05 11:49)
Sipi: Sipi Source Git-Version: 454d1d1
Sipi: Sipi Source Git-Version: 454d1d1
Sipi: Cache at "/sipi/cache" cachesize=104857600 nfiles=0 hysteresis=0.100000
Sipi: Cache at "/sipi/cache" cachesize=104857600 nfiles=0 hysteresis=0.100000
Sipi: Sipi server starting
Sipi: Sipi server starting
Sipi: Serving images from /sipi/images
Sipi: Serving images from /sipi/images
Sipi: Starting shttps server with 8 threads
Sipi: Starting shttps server with 8 threads
Sipi: Added route /make_thumbnail with script /sipi/scripts/make_thumbnail.lua
Sipi: Added route /make_thumbnail with script /sipi/scripts/make_thumbnail.lua
Sipi: Added route /convert_from_binaries with script /sipi/scripts/convert_from_binaries.lua
Sipi: Added route /convert_from_binaries with script /sipi/scripts/convert_from_binaries.lua
Sipi: Added route /convert_from_file with script /sipi/scripts/convert_from_file.lua
Sipi: Added route /convert_from_file with script /sipi/scripts/convert_from_file.lua
Sipi: Added route /Knora_login with script /sipi/scripts/Knora_login.lua
Sipi: Added route /Knora_login with script /sipi/scripts/Knora_login.lua
Sipi: Added route /Knora_logout with script /sipi/scripts/Knora_logout.lua
Sipi: Added route /Knora_logout with script /sipi/scripts/Knora_logout.lua
Sipi: Server listening on port 1024
Sipi: Server listening on port 1024
```

Comment (subotic) [reply docs/src/paradox/07-sipi/setup-sipi-for-knora.md]: Yes, because the way logging is set up in sipi.

Comment (subotic) [reply docs/src/paradox/07-sipi/setup-sipi-for-knora.md]: release notes updated

**Comment by subotic** [comment]
Thank you :-)

**Pull request merged**